DistFiles/xMatter/bloom-xmatter-common.less

```diff
@@ -20,6 +20,9 @@
          //Just center the margin box, (for now, we're ignoring the binding) 
           left: @MarginOuter;
           width:  @PageWidth - (2 * @MarginOuter);
+          IMG { /* BL-2353 Keeps XMatter thumb images from going too wide (or too small) */
+              max-width: @PageWidth - (2 * @MarginOuter);
```

Comment:
The comment on line 20 is pre-existing:
//Just center the margin box, (for now, we're ignoring the binding)
Probably because it's not in publishmode yet.